contrib/prometheus-protobufs/prompb/remote.proto

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package prometheus;
+
+option go_package = "prompb";
+
+import "prompb/types.proto";
+import "gogoproto/gogo.proto";
```

Comment:
Here `remote.proto` from the `Prometheus` repository depends on `gogo.proto` from the [Protocol Buffers for Go with Gadgets](https://github.com/cosmos/gogoproto) repository. That repository is quite big and I need only one file from it again, so I just downloaded it into our `contrib/prometheus-protobufs-gogo` folder instead of making a new submodule.